crates/worker/src/partition_processor_manager.rs

```diff
@@ -467,14 +471,17 @@ impl<T: TransportConnect> PartitionProcessorManager<T> {
         let shutdown = cancellation_watcher();
         tokio::pin!(shutdown);
 
+        let (event_tx, mut event_rx) = mpsc::unbounded_channel();
+        let (effect_tx, mut effect_rx) = mpsc::unbounded_channel();
```

Comment:
Event and effect are quite generic. Are the more descriptive names for those channels, maybe?

Reply:
I assume that those channels don't have to be unbounded, right? Maybe we add a limit to avoid too excessive resource consumption.

Reply:
I was worried about using bound channels in case (for some reason) there were lots of updates (and possibly lots of effects/evetns pushed on the channel) that can cause a dead lock (loop can't processes the queues because it's trying to push more objects to the queue which is already full)

So I thought it should be safer to use unbound channel in that case to avoid this deadlock and all messages should get processed eventually even if for a temporary spike in memory usage.

Reply:
Are both being pushed to from the main event loop?

Reply:
No, only the effects actually are pushed to from the main event loop (seems I am not fully back from vacation 😅). Events then can become a bound channel then.